augly/image/__init__.py

```diff
@@ -173,6 +181,8 @@
     "contrast",
     "convert_color",
     "crop",
+    "distort_barrel",
+    "distort_pincushion",
```

Comment:
Add "distort_barrel_intensity" & "distort_pincushion_intensity" in this list too, i.e. on line 220!